Mage.Sets/src/mage/cards/c/ConsumingFerocity.java

```diff
@@ -51,7 +52,10 @@ public ConsumingFerocity(UUID ownerId, CardSetInfo setInfo) {
         this.addAbility(new SimpleStaticAbility(Zone.BATTLEFIELD, new BoostEnchantedEffect(1, 0, Duration.WhileOnBattlefield)));
 
         // At the beginning of your upkeep, put a +1/+0 counter on enchanted creature. If that creature has three or more +1/+0 counters on it, it deals damage equal to its power to its controller, then destroy that creature and it can't be regenerated.
-        this.addAbility(new BeginningOfUpkeepTriggeredAbility(new ConsumingFerocityEffect(), TargetController.YOU, false));
+        Effect effect = new ConsumingFerocityEffect();
```

Comment:
could just inline this `new ConsumingFerocityEffect()` when you add it to the ability, rather than declaring it